packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -65,13 +64,17 @@ const CurrencySwitcher = () => {
             <div className='flex-none '>{IconToCurrencyMapper[iconCurrency].icon}</div>
             <div className='grow'>
                 <Text
-                    bold={isDemo}
                     className={isDemo ? 'text-status-light-information' : 'text-system-light-less-prominent-text'}
                     size='sm'
+                    weight={isDemo ? 'bold' : 'normal'}
                 >
                     {isDemo ? activeAccount.display_balance : IconToCurrencyMapper[iconCurrency].text}
                 </Text>
-                <Text bold={!isDemo} className={!isDemo ? 'text-status-light-success' : undefined} size='sm'>
+                <Text
+                    className={!isDemo ? 'text-status-light-success' : undefined}
```

Comment:
for this I suggest to use wrap it inside of `qtMerge(!isDemo ? 'text-status-light-success': '')`. qtMerge will eliminate the repeated classes and cleanup any unnecessary classes.